lib/actions/up.js

```diff
@@ -33,20 +35,35 @@ module.exports = async (db, client) => {
       throw error;
     }
 
-    const { changelogCollectionName } = await config.read();
-    const changelogCollection = db.collection(changelogCollectionName);
+    if (item.appliedAt != "CUSTOM")
+    {
+      // console.log("writing", item.appliedAt);
+      const { changelogCollectionName } = await config.read();
+      const changelogCollection = db.collection(changelogCollectionName);
 
-    const { fileName } = item;
-    const appliedAt = new Date();
+      const { fileName } = item;
+      const appliedAt = new Date();
 
-    try {
-      await changelogCollection.insertOne({ fileName, appliedAt });
-    } catch (err) {
-      throw new Error(`Could not update changelog: ${err.message}`);
+      try {
+        await changelogCollection.insertOne({ fileName, appliedAt });
+      } catch (err) {
+        throw new Error(`Could not update changelog: ${err.message}`);
+      }
     }
+
     migrated.push(item.fileName);
   };
 
-  await pEachSeries(pendingItems, migrateItem);
+  // console.log("some", global.options);
+  if (global.options.custom) {
+    await migrateItem(
+      { fileName: global.options.custom, appliedAt: "CUSTOM" },
```

Comment:
@bdcorps IMO it would be better if we stored the date in appliedAt field. It will be useful to check when this particular migration was applied. Perhaps a boolean field will be better to differentiate between the regular and custom migration.